docs/flows-format.adoc

```diff
@@ -0,0 +1,250 @@
+// Automatically generated by 'hack/asciidoc-flows-gen.sh'. Do not edit.
+:_content-type: REFERENCE
+[id="network-observability-flows-format-specifications_{context}"]
+= Network Flows format specifications
+
+Description::
++
+This is the specification of the network flows format, used both internally and in exporter feature such as Kafka export.
```

Comment:
```suggestion
This is the specification of the network flows format, used both internally and when exporting flows to Kafka.
```

Reply:
Also one question, when you refer to using internally, are you referring to using to query Loki?